### PR TITLE
Fix crawling throwing and turret shooting

### PR DESCRIPTION
**Pull request by BombasterDS**
## About the PR
Fix crawl fix throwing bug 
Fixed turret can't shoot lying people
Resolves: #656

**Changelog**

:cl:
- fix: Fixed throwing objects collision with tables
- fix: Fixed turrets couldn't shoot lying people 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new property, `IgnoreThrow`, allowing certain entities to ignore projectile throw actions.
	- Enhanced collision prevention logic for projectiles, improving gameplay interactions.
	- Added functionality for targeted shooting in the gun system, enhancing shooting accuracy.

- **Bug Fixes**
	- Adjustments made to projectile targeting mechanics, particularly for crawling gameplay.

- **Configuration Changes**
	- Updated entity configurations to include `ignoreThrow: true` for specific furniture types, affecting how they interact with thrown projectiles.

### CODE REVIEW

**Comment by Aineias1** [comment]
**.**

**Comment by Aineias1** [comment]
Fuck

**Comment by Aineias1** [comment]
We need this badly i think

**Review by whateverusername0 — APPROVED** [review]

File: Content.Server/NPC/Systems/NPCCombatSystem.Ranged.cs [hunk 1]
```diff
@@ -202,7 +202,7 @@ private void UpdateRanged(float frameTime)
                 return;
             }
 
-            _gun.AttemptShoot(uid, gunUid, gun, targetCordinates);
+            _gun.AttemptShoot(uid, gunUid, gun, targetCordinates, comp.Target);
```
[the remark this answers is not in the document]
Comment (BombasterDS) [reply Content.Server/NPC/Systems/NPCCombatSystem.Ranged.cs:205]: damn you stupid ai glados scp-079!!! >:(!!! I need to take target from component and not from events! MORON!!!!!

**Review by Piras314 — APPROVED** [review]

**Pull request merged**